ui/ui-frontend/projects/referential/src/app/ontology/ontology-preview/schema-information-tab/schema-information-tab.component.html

```diff
@@ -0,0 +1,159 @@
+<form [formGroup]="form" class="gap-5">
+  <div class="gap-3 w-100">
```

Comment:
Tu peux ajouter "align-items-stretch" et supprimer tous les `class="w-100"`
```suggestion
  <div class="gap-3 w-100 align-items-stretch">
```